flake.nix

```
@@ -0,0 +1,30 @@
+{
+  description = "wander: a terminal app/TUI for HashiCorp Nomad";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let pkgs = nixpkgs.legacyPackages.${system};
+      in {
+        packages.default = self.packages.${system}.wander;
+
+        packages.wander = pkgs.buildGoModule {
+          name = "wander";
+          vendorHash = "sha256-SqDGXV8MpvEQFAkcE1NWvWjdzYsvbO5vA6k+hpY0js0=";
+          src = ./.;
+
```

```suggestion
          src = ./.;
         
          ldflags =
            [ "-X github.com/robinovitch61/wander/cmd.Version=v0.14.0" ];
```

The way to plug in version would be to add it to the build spec like above. I don't think there is any way to plug this in dynamically.

By default, `nix flake` just grabs whatever commit `main` is on, and pins that version in `flake.lock`. If we have a workflow to keep bumping this on releases, the actual code would but the release tag and commits on top.

How do you feel about it? The other option is to set the version for flake to something static like, `v{version}-flake` or just `flake`.

---

ping @robinovitch61

---

Ah sorry for the delay, for some reason wasn't getting emails about this! Gonna merge as is and see what I can do about the version embed later, thanks so much for this :)